src/etwlogger.man

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
```

Comment:
I have obviously thought that it needs to stay under "src" since it gets compiled by "mc"... Not sure what is asked of me to do here... 

---
In reply to: [126215361](https://github.com/Azure/azure-c-shared-utility/pull/89#discussion_r126215361) [](ancestors = 126215361)